src/software/guides/concepts.md

```diff
@@ -473,6 +467,112 @@ statusDatas[i].device = deviceLookup[statusDatas[i].device.id];
+| IsIncluded | A boolean, which if `true`, will include the properties of a given [Entity]({{site.baseurl}}/sdk/software/api/reference/#Entity) type defined in `Fields` for the entities of the result set. Otherwise, if this boolean is false, the properties defined in `Fields` will be excluded.
+
+### Examples
+
+A simple [example]({{site.baseurl}}/software/api/runner.html#sample:get-lightweight-device-response) of this can be illustrated by using the property selector with `Device`. The `Device` object can have many properties which may not be useful to all use-cases. For example, if I have an add-in to display a list of 500 devices by name. We only want our `Device` objects to have the properties `Name` and `Id`, so we set our `PropertySelector` object like so:
+
+#### Javascript
+
+##### Request
+
+```javascript
+api.call("Get", {
+  "typeName": "Device",
+  "propertySelector":
+  {
+      fields: ["id", "name"],
+      isIncluded: true
+  },
+  "resultsLimit": 500
+}, function(result) {
+  console.log("Done: ", result);
+}, function(e) {
+  console.error("Failed:", e);
+});
+```
+
+##### Response
+
+```json
+[
+    {
+        "name": "Work Truck 10",
+        "id": "b1"
+    },
+    {
+        "name": "Delivery Van 6",
+        "id": "b2"
+    }
+]
+```
+
+In our example, making this call using the property selector results in the total JSON size over the wire of 5.4 kB and time of 45 ms.
+
+Making the same call, without property selector (returning all properties) results in 41.8 kB of JSON sent over the wire and a round trip time of 320 ms.
+
+| using property selector |device count| size | time |
+|--|--|--|--|
+|false|500|41.8 kB|320 ms|
+|true|500|5.4 kB|45 ms|
+|--|--|--|--|
+|Improvement||**-36.4 kB**|**-275 ms**|
+
+#### C# Example
+
+```csharp
+var results = await api.CallAsync<List<Device>>(
+  "Get",
+  typeof(Device),
+  new
+  {
+    propertySelector = new PropertySelector
+    {
+        Fields = new List<string>
+        {
+          nameof(Device.Name),
+          nameof(Device.Id)
+        },
+        IsIncluded = true
+    },
+    resultsLimit = 500
+  });
+```
+
+## List of Supported Entities
+
+Below is a list of entities that are support the PropertySelector functionality.
+
+| **Entity** | **Supported in Release** | **Notes**
+| --- | --- | -- |
+| [Device]({{site.baseurl}}/software/api/reference/#Device) | 8.0 | The following properties are not supported: `deviceFlags`, `isAuxInverted`, `deviceType`, `productId`, `autogroups`, `auxWarningSpeed`, `enableAuxWarning`
```

Comment:
Same here. Should we specify it only works with search and then update later once the Get is fixed?